src/index.ts

```diff
@@ -281,15 +259,23 @@ async function runCumulusTask(taskFunction, cumulusMessage,
     cmaStdin.write('\n<EOC>\n');
     const createNextEventOutput = await getCmaOutput(rl, errorObj);
     cmaStdin.write('\n<EXIT>\n');
+    if (isLoadNestedEventInput(createNextEventOutput)) {
+      throw new Error(`Invalid typing recieved from
+      createNextEventOutput: ${JSON.stringify(createNextEventOutput)}`);
+    }
     return createNextEventOutput;
-  }
-  catch (error) {
+  } catch (error) {
     if (error.name && error.name.includes('WorkflowError')) {
+      const returnObject = cumulusMessage;
+      returnObject.payload = null;
+      returnObject.exception = error.name;
       return { ...cumulusMessage, payload: null, exception: error.name };
     }
     throw error;
   }
 }
 
+
 exports.runCumulusTask = runCumulusTask;
```

Comment:
If the functions are exported when they are declared, these export statements are not needed.

Reply:
f4b3013